third_party/2and3/yaml/__init__.pyi

```diff
@@ -22,7 +22,7 @@ def emit(events, stream=..., Dumper=..., canonical=..., indent=..., width=..., a
 def serialize_all(nodes, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
 def serialize(node, stream=..., Dumper=..., **kwds): ...
 def dump_all(documents: Sequence[Any], stream: IO[str] = ..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> Any: ...
-def dump(data: Any, stream: IO[str] = ..., Dumper=..., **kwds) -> Any: ...
```

Comment:
You removed a bunch of annotations; that doesn't seem correct. I think we just need an `Optional` around the `IO[str]`.

Reply:
Sure. Will add them back.
Do we not need `None`?

Reply:
`Optional` is short for a `Union` with None, so it means that `None` is allowed to be passed in.

Reply:
Sure. On it.